Products/Zuul/facades/devicefacade.py

```diff
@@ -841,7 +853,7 @@ def getComponentGraphs(self, uid, meta_type, graphId, allOnSame=False):
         # get the graph def
         for comp in components:
             # find the first instance
-            for graph in comp.getGraphObjects():
+            for graph, ctx in comp.getGraphObjects():
```

Comment:
It appears that here you do assume that `getGraphObjects` returns a sequence of two-element tuples here, *unlike* in earlier code (line 737).